api/spec/v1.yaml

```diff
@@ -2572,6 +2596,60 @@ components:
             the `/{runtime}/accounts/{address}` endpoint.
           example: false
 
+    EvmNftList:
+      allOf:
+        - $ref: '#/components/schemas/List'
+        - type: object
+          required: [evm_nfts]
+          properties:
+            evm_nfts:
+              type: array
+              items:
+                $ref: '#/components/schemas/EvmNft'
+              description: A list of L2 EVM NFT (ERC-721, ...) instances.
+          description: A list of NFT instances.
+
+    EvmNft:
+      type: object
+      required:
+        - contract_addr
+        - id
+      properties:
+        contract_addr:
+          type: string
+          description: The Oasis address of this NFT's contract.
+          example: 'oasis1qp2hssandc7dekjdr6ygmtzt783k3gn38uupdeys'
+        eth_contract_addr:
+          type: string
+          description: The Ethereum address of this token's contract.
+          example: *eth_address_1
+        token_name:
+          type: string
+        token_symbol:
+          type: string
+        token_type:
+          $ref: '#/components/schemas/EvmTokenType'
+        token_total_supply:
+          <<: *BigIntType
+        id:
+          <<: *BigIntType
+        metadata_uri:
+          type: string
+        metadata_accessed:
+          type: string
+        name:
+          type: string
+          description: Identifies the asset to which this NFT represents
```

Comment:
Nit here and a few lines down: No "to"

Reply:
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md?plain=1#L206

can you file upstream :pray:

Reply:
https://github.com/ethereum/EIPs/commit/223e46c7ce39de62c54bb6a2f9d1e630d3aff079#diff-93c4038537e3c4794bc42d0cd3e75d12876b722a743e0932c28549b23ea28edeL208 interesting, the "to" comes from when it was "DAR" that granted access to an asset

Reply:
huh, but even back then, shouldn't it have been ownership "of" an asset? :thinking:

Reply:
Filed https://github.com/ethereum/EIPs/pull/7795. We can fix here independently tho.

Reply:
I added a commit to fix it here. too bad about the lint hell in the EIPs repository